velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -139,6 +139,13 @@ bool SubstraitToVeloxPlanValidator::validateScalarFunction(
   if (name == "extract") {
     return validateExtractExpr(params);
   }
+  if (name == "regexp_extract_all" &&
+      !scalarFunction.arguments()[1].value().has_literal()) {
+    std::cout
```

Comment:
Nit: can we use `VLOG(1) <<` to print the message like other place? It has a better control to avoid verbose log.